### PR TITLE
(Ready) Add translation for Event/GuildTutorial.txt

### PR DESCRIPTION
**Pull request by Noitcereon**
## Changes
- Added a machine translated GuildTutorial.txt with some minor corrections.

### CODE REVIEW

**Comment by Giratena** [comment]
Ooh thank you for your PR @Noitcereon!

The translation is a bit Engrishy, but since this is quite a short file I should be able to go through at some point with better translations to show you what I mean when I get around to reviewing this.

**Comment by Giratena** [comment]
I've also edited your pr title to (Ready) for you as this is what we are using to show PRs that need to be looked at ^^

**Comment by Noitcereon** [comment]
I tried to make it less "Engrishy" (make it feel more like natively spoken English). Hopefully it's better.

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-はい。ギルドのカードマスターにはランクがあり、\n上からA、B、C、D、E、Fと6段階あります。,
-例えば、あなたの師匠であるコノハさんは\nB級のカードマスターですね。B級ともなると\n「守護者」として地域の治安維持を任されることもあります。,
-さて、あなたはこれからギルドで任務をこなし、\n少しづつランクをあげていくわけですが…。,
-非常に良いタイミングでギルドに登録されましたね。\n今この街では10年に1度の大きな大会が開かれています。,
-大会の最中は他のカードマスターに勝つことで、\nどんどん自分のランクをあげていくことができるんですよ。,
-B級やA級になるためにはふつう何年もかかりますが、\nうまく勝ち抜ければこの大会中にA級になることだって夢ではありません。,
-F級から一気にA級まで行った人はわたしの知る限りではいませんが…\nもしかしたらあなたならできるかもしれません。頑張ってくださいね！,
-ありがとうございます！,
-ギルドメンバーになった証として、ギルドリングを手に入れた！,
+スライムのサキュバスを倒した次の日。僕は師匠に言われた通りカードマスターギルドの建物を訪ねていた。,The day after I defeated the slime succubus; I went to the Card Master Guild building as my master had told me to.
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Event/GuildTutorial.txt:2]: Hey man, just a heads up, for all instances in this translation, 'Cardmaster' or 'cardmaster' is always one word.

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+あら、見ない顔ですね。新規の登録者さんですか？,Oh; I don't recognize you. Are you a new card master?
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Event/GuildTutorial.txt:3]: 'cardmaster'

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+リーフさんという方に会いに来ました。,I came to see a man named Leif.
+ああ。あなたがコノハさんのお弟子さんですね。,Ah. You're Konoha's student, aren't you?
+あいにくギルドマスターのリーフは外出中ですが、\n言伝を受けております。,Unfortunately; Guild Master Leif is out, but\n he has been informed.
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Event/GuildTutorial.txt:6]: maybe change 'been informed' to 'received word' to help connect the fact that they got a note or letter.

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
+「南の村の守護者コノハが、\n若いけれど優秀な弟子をギルドに向かわせる」\nとのことでしたが…。,"Konoha; the protector of the southern village; will be sending a young but talented apprentice to the guild;" it said.
+ふふ、確かになかなかの魔力ですね。\nあなたの才能がこれから伸びていくのが楽しみです。,Fufu; that's certainly some impressive magical power.\nI look forward to seeing your talent develop even further.
+さて、手紙の内容によれば、\nギルドへの登録を希望されているんですよね？,Now; according to the contents of the letter;\n you wish to register with the guild, right?
+今回は守護者資格のあるカードマスターからの推薦ですので、\n入会審査はなしで大丈夫です。,Since you have a recommendation from a card master who has guardian qualifications; there is no need for an entrance examination.
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Event/GuildTutorial.txt:10]: 'cardmaster'

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 5]
```diff
@@ -1,24 +1,24 @@
...
+さっそく書類に必要事項をお書きください。,Please fill out the necessary information on this form.
+書き終わりましたか？ありがとうございます。,Are you done? Thank you!
+えーと、お名前は{主人公}さん……、\nふむふむ、出身はだいぶ北にある村なんですね。,Umm; so your name is {主人公}... \n I see; you're from a village quite far to the north.
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Event/GuildTutorial.txt:13]: Might flow better if you change it to 'I see. You're' with a period

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 6]
```diff
@@ -1,24 +1,24 @@
...
+登録おつかれさまです。\n新人カードマスターはまずはFランクからのスタートです。,Thank you for registering. \n New Card Masters will start out at F rank.
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Event/GuildTutorial.txt:14]: 'cardmaster' here should be lowercase

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 7]
```diff
@@ -1,24 +1,24 @@
...
+Fランクですか？,F rank?
+はい。ギルドのカードマスターにはランクがあり、\n上からA、B、C、D、E、Fと6段階あります。,Yes. Guild card masters have ranks; with six levels: A; B; C; D; E; and F from the top.
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Event/GuildTutorial.txt:16]: 'cardmaster.' Might sound better if you add 'top to bottom.'

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 8]
```diff
@@ -1,24 +1,24 @@
...
+例えば、あなたの師匠であるコノハさんは\nB級のカードマスターですね。B級ともなると\n「守護者」として地域の治安維持を任されることもあります。,For example; your master; Konoha; is a B-rank card master. B-rank card masters are sometimes entrusted with maintaining peace in the area as "guardians."
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Event/GuildTutorial.txt:17]: 'cardmaster'

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 9]
```diff
@@ -1,24 +1,24 @@
...
+さて、あなたはこれからギルドで任務をこなし、\n少しづつランクをあげていくわけですが…。,As you complete missions for the guild; you will gradually increase your rank...
+非常に良いタイミングでギルドに登録されましたね。\n今この街では10年に1度の大きな大会が開かれています。,You have joined the guild at an extremely opportune time.\nRight now; a major tournament that takes place once every ten years is being held in this city.
+大会の最中は他のカードマスターに勝つことで、\nどんどん自分のランクをあげていくことができるんですよ。,During the tournament; you can steadily increase your rank by defeating other card masters.
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Event/GuildTutorial.txt:20]: 'cardmaster'

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 10]
```diff
@@ -1,24 +1,24 @@
...
+B級やA級になるためにはふつう何年もかかりますが、\nうまく勝ち抜ければこの大会中にA級になることだって夢ではありません。,It usually takes years to reach B or A rank; but if you win your battles; it's not a dream to reach A rank during this tournament.
+F級から一気にA級まで行った人はわたしの知る限りではいませんが…\nもしかしたらあなたならできるかもしれません。頑張ってくださいね！,As far as I know; no one has gone from F-class to A-class in one go... but maybe you can do it. Good luck!
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Event/GuildTutorial.txt:22]: Maybe change 'class' to 'rank' for consistency

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 11]
```diff
@@ -1,24 +1,24 @@
...
+登録おつかれさまです。\n新人カードマスターはまずはFランクからのスタートです。,Thank you for registering. \n New Card Masters will start out at F rank.
+Fランクですか？,F rank?
+はい。ギルドのカードマスターにはランクがあり、\n上からA、B、C、D、E、Fと6段階あります。,Yes. Guild card masters have ranks; with six levels: A; B; C; D; E; and F from the top.
+例えば、あなたの師匠であるコノハさんは\nB級のカードマスターですね。B級ともなると\n「守護者」として地域の治安維持を任されることもあります。,For example; your master; Konoha; is a B-rank card master. B-rank card masters are sometimes entrusted with maintaining peace in the area as "guardians."
+さて、あなたはこれからギルドで任務をこなし、\n少しづつランクをあげていくわけですが…。,As you complete missions for the guild; you will gradually increase your rank...
+非常に良いタイミングでギルドに登録されましたね。\n今この街では10年に1度の大きな大会が開かれています。,You have joined the guild at an extremely opportune time.\nRight now; a major tournament that takes place once every ten years is being held in this city.
+大会の最中は他のカードマスターに勝つことで、\nどんどん自分のランクをあげていくことができるんですよ。,During the tournament; you can steadily increase your rank by defeating other card masters.
+B級やA級になるためにはふつう何年もかかりますが、\nうまく勝ち抜ければこの大会中にA級になることだって夢ではありません。,It usually takes years to reach B or A rank; but if you win your battles; it's not a dream to reach A rank during this tournament.
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Event/GuildTutorial.txt:21]: Maybe 'fantasy' instead of 'dream,' or just something like 'it's not impossible.'

**Review by HotHeadEmperor — COMMENTED** [review]
Wanted to take a stab at a few quick corrections and recommendations.

**Comment by Noitcereon** [comment]
Force pushed to fix minor mistake in previous commit.

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 12]
```diff
@@ -1,24 +1,24 @@
...
-B級やA級になるためにはふつう何年もかかりますが、\nうまく勝ち抜ければこの大会中にA級になることだって夢ではありません。,
-F級から一気にA級まで行った人はわたしの知る限りではいませんが…\nもしかしたらあなたならできるかもしれません。頑張ってくださいね！,
-ありがとうございます！,
-ギルドメンバーになった証として、ギルドリングを手に入れた！,
+スライムのサキュバスを倒した次の日。僕は師匠に言われた通りカードマスターギルドの建物を訪ねていた。,The day after I defeated the slime succubus; I went to the cardmaster guild building as my master had told me to.
+あら、見ない顔ですね。新規の登録者さんですか？,Oh; I don't recognize you. Are you a new cardmaster?
+リーフさんという方に会いに来ました。,I came to see someone called Leaf.
+ああ。あなたがコノハさんのお弟子さんですね。,Ah. You're Konoha's student, aren't you?
+あいにくギルドマスターのリーフは外出中ですが、\n言伝を受けております。,Unfortunately; Guild Master Leaf is out, but\n she has received word of your coming.
+「南の村の守護者コノハが、\n若いけれど優秀な弟子をギルドに向かわせる」\nとのことでしたが…。,"Konoha; the protector of the southern village; will be sending a young but talented apprentice to the guild;" it said.
```
Comment (Giratena) [inline CSV/Language/English/Event/GuildTutorial.txt:7]: "It said" is a little unnatural for English! Maybe...

She was told; "Konoha; the protector of the southern village; will be sending a young but talented apprentice to the guild."

File: CSV/Language/English/Event/GuildTutorial.txt [hunk 13]
```diff
@@ -1,24 +1,24 @@
...
+ふふ、確かになかなかの魔力ですね。\nあなたの才能がこれから伸びていくのが楽しみです。,Fufu; that's certainly some impressive magical power.\nI look forward to seeing your talent develop even further.
+さて、手紙の内容によれば、\nギルドへの登録を希望されているんですよね？,Now; according to the contents of the letter;\n you wish to register with the guild, right?
+今回は守護者資格のあるカードマスターからの推薦ですので、\n入会審査はなしで大丈夫です。,Since you have a recommendation from a cardmaster who has guardian qualifications; there is no need for an entrance examination.
+さっそく書類に必要事項をお書きください。,Please fill out the necessary information on this form.
+書き終わりましたか？ありがとうございます。,Are you done? Thank you!
+えーと、お名前は{主人公}さん……、\nふむふむ、出身はだいぶ北にある村なんですね。,Umm; so your name is {主人公}... \n I see. You're from a village quite far to the north.
```
Comment (Giratena) [inline CSV/Language/English/Event/GuildTutorial.txt:13]: Umm -> Let's see. So your name is

Also we aren't included /n in the translation (with some exceptions). So feel free to remove them ^^

**Review by Giratena — CHANGES_REQUESTED** [review]
Much much better! Nice work @HotHeadEmperor for proofreading it and shaping it up and @Noitcereon being so quick on the changes.

Two comments from me. Once they are done I'll merge ^^

**Review by Giratena — APPROVED** [review]
Nicely done! I will approve and merge this in ^^

**Pull request merged**